styles/components/featured-resources.css

```diff
@@ -50,36 +50,36 @@
   align-items: flex-start;
   display: grid;
   grid-gap: 0 30px;
-  grid-template-areas: 'image category'
-    'image title'
-    'image excerpt';
+  grid-template-areas: 'image copy';
 
   &__image {
     border: solid 2px #fff;
     display: block;
     grid-area: image;
-    height: 120px;
```

Comment:
Another liberty I took – adjusted the images to take a little less space so that line lengths read a little more reasonably.